src/js/views/forms/form/form.scss

```diff
@@ -189,6 +189,30 @@
   font-size: 12px;
 }
 
+.form__locked {
+  align-items: center;
+  border-left: 1px solid $black-80;
+  display: flex;
+  margin-left: 20px;
+  padding-left: 20px;
+}
+
+.form__locked-icon {
+  font-size: 20px;
+  margin-right: 12px;
+
+  .icon {
+    color: $black-60;
+    font-size: 20px;
+    vertical-align: -0.125em;
+  }
+}
+
+.form__locked-text {
+  font-size: 12px;
+  width: 160px;
```

Comment:
With `width: 160px;`:

<img width="420" alt="Screenshot 2023-10-27 at 4 11 57 PM" src="https://github.com/RoundingWell/care-ops-frontend/assets/35355575/2dc37f3a-db5a-436b-a37f-ded9927a2019">

Without:
<img width="463" alt="Screenshot 2023-10-27 at 4 41 21 PM" src="https://github.com/RoundingWell/care-ops-frontend/assets/35355575/29514ae4-3bb9-478a-a54a-951aef736d85">

Reply:
so the other messages just use two separate text lines.  that'd be an option as well.